src/cmds/cloud/createThing.js

```diff
@@ -1,39 +1,21 @@
 /* eslint-disable no-console */
 import yargs from 'yargs';
+import chalk from 'chalk';
 import { Client } from '@cesarbr/knot-cloud-sdk-js';
 import options from './utils/options';
 import getFileCredentials from './utils/getFileCredentials';
 
-const createThing = (args) => {
+const createThing = async (args) => {
   const client = new Client({
-    protocol: args.protocol,
     hostname: args.server,
     port: args.port,
-    pathName: args.pathName,
-    id: args['client-id'],
-    token: args['client-token'],
+    protocol: args.protocol,
+    token: args.token,
   });
```

Comment:
`username` and `password` are missing here.

Reply:
hm?

Reply:
We use the `token` to provision things, not `username` and `password` :eyes:

Reply:
Yeah. In this case, they're RabbitMQ credentials.